docs/source/workflow_outputs.rst

```diff
@@ -791,13 +793,10 @@ Electric panel loads, as well as calculated total loads and capacities for each
   Electric Panel Load: Other (W)                                                    Sum of other loads
   Electric Panel Load: <Type>: Total Load (W)                                       Calculated NEC total load
   Electric Panel Load: <Type>: Total Capacity (A)                                   Total panel capacity
```

Comment:
Should this note be something closer to "Calculated NEC total capacity"? (I.e., it isn't the panel's maximum current rating -- it is the Total Load (W) divided by panel voltage.)